client/handler.go

```diff
@@ -230,13 +233,17 @@ func (h *Handler) List(w http.ResponseWriter, r *http.Request, ps httprouter.Par
 //
 //     Responses:
 //       200: oAuth2Client
+//		 401: genericError
 //       404: genericError
 //       500: genericError
 func (h *Handler) Get(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
 	var id = ps.ByName("id")
 
 	c, err := h.r.ClientManager().GetConcreteClient(r.Context(), id)
 	if err != nil {
+		if errors.Cause(err).Error() == sqlcon.ErrNoRows.ErrorField {
```

Comment:
I believe this should be enough:

```suggestion
		if errorsx.Cause(err) == sqlcon.ErrNoRows {
```